src/Value/Resolver.php

```diff
@@ -28,20 +30,22 @@ public function set($model, $attribute, $groups)
     /**
      * get the field's value
      *
-     * @param  mixed  $resource
+     * @param  mixed  $model
      * @param  string  $attribute
-     * @param  Whitecube\NovaFlexibleContent\Layouts\Collection  $layouts
-     * @return Illuminate\Support\Collection
+     * @param  \Whitecube\NovaFlexibleContent\Layouts\Collection  $layouts
+     * @return \Illuminate\Support\Collection|null
      */
-    public function get($resource, $attribute, $layouts)
+    public function get($model, $attribute, $layouts)
     {
+        $resource = $model; // to avoid Named Arguments issue but keep using better var name;
+
```

Comment:
Same here, why would `$model` be better than `$resource` for named arguments? This argument is `mixed` and should therefore not be confused with a model (it could be another Layout instance, or any other container object depending on the project).